content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -0,0 +1,199 @@
+    statsd.WithMaxMessagesPerPayload(256), // sets the maximum number of messages in a single datagram
+)
+// All metrics sent from this instance of the client will be buffered and sent in
+// packets containing several metrics
+client.Gauge("kafka.health", 1, []string{"env:production", "partition:1", "partition:2"}, 1)
+```
+{{% /tab %}}
+
+{{% tab "Python" %}}
+Using Datadog's official Python library [datadogpy](https://github.com/DataDog/datadogpy):
+
+```python
+# Load the dogstats module.
+from datadog import DogStatsd
+
+# All metrics will be buffered and sent in one packet when the block completes
+with DogStatsd(host="127.0.0.1", port=8125, max_buffer_size=25) as batch:
+    batch.gauge('users.online', 123)
+    batch.gauge('active.connections', 1001)
+```
+{{% /tab %}}
+
+{{% tab "Ruby" %}}
+Using Datadog's official Ruby library [dogstatsd-ruby](https://github.com/DataDog/dogstatsd-ruby):
+
+```ruby
+require 'datadog/statsd'
+
+# Create a stats instance
+statsd = Datadog::Statsd.new('127.0.0.1', 8125)
+
+# All metrics will be buffered and sent in one packet when the block completes
+statsd.batch do |s|
+  s.increment('page.views')
+  s.gauge('users.online', 123)
+end
+```
+{{% /tab %}}
+
+{{% tab "Java" %}}
+Using Datadog's official Java library [java-dogstatsd-client](https://github.com/DataDog/java-dogstatsd-client):
+
+```java
+// Create a stats instance with a buffer size of 256
+private static final StatsDClient statsd = new NonBlockingStatsDClient("namespace", "127.0.0.1", 8125, 256);
+
+// All metrics sent from this instance of the client will be buffered and sent in
+// packets containing several metrics
+public void foo() {
+    statsd.incrementCounter("foo");
+    statsd.recordGaugeValue("bar", 100);
+}
+```
+{{% /tab %}}
+
+{{% tab "C#" %}}
+Using Datadog's official C# library [dogstatsd-csharp-client](https://github.com/DataDog/dogstatsd-csharp-client):
+
+```csharp
+// Create the transport
+StatsdUDP udp = new StatsdUDP("127.0.0.1", 8125);
+
+using (udp)
+{
+    // Create a stats instance with "udp" as transport
+    Statsd s = new Statsd(udp);
+
+    s.Add<Statsd.Counting,int>("stat-name", 1);
+    s.Add("event title", "content", priority: "low");
+    s.Add<Statsd.Counting,int>("stat-name", 1);
+
+    // All metrics buffered before this call will be sent in one packet
+    s.Send();
+}
+```
+{{% /tab %}}
+
+{{% tab "PHP" %}}
+Using Datadog's official PHP library [php-datadogstatsd](https://github.com/DataDog/php-datadogstatsd):
+
+```php
+require __DIR__ . '/vendor/autoload.php';
+
+use DataDog\BatchedDogStatsd;
+
+$client = new BatchedDogStatsd();
+
+// All metrics sent from this instance of the client will be buffered and sent in
+// packets containing several metrics
+$client->increment('your.data.point');
+$client->increment('your.data.point', .5);
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Sample your metrics
+
+It is possible to reduce the traffic from your DogStatsD client and the Agent by setting a sample
```

Comment:
Either:
`from your DogStatsD client to the Agent`
or
`between your DogStatsD client and the Agent`